test/helpers/decodeEvent.js

```diff
@@ -0,0 +1,13 @@
+const abi = require('web3-eth-abi')
+
+module.exports = {
+    decodeEventsOfType: (receipt, eventAbi) => {
```

Comment:
Wow, this is really nice! Way better than what I was doing [here](https://github.com/aragon/dao-kits/blob/master/kits/multisig/test/multisig.js#L22). I wonder if we could put it under `@aragon/test-helpers` or some other place which makes it easy for reuse.
I also think it may be nice to add a wrapper with `abi.filter(abi => abi.name === eventName && abi.type === 'event')[0]`, so just passing the receipt, the global abi and the event name is enough.

Reply:
Yes... we should migrate to `@aragon/test-helpers` :D.

Will change the API to take in the abi too.